### ISSUE TITLE
Board Coordinates too big

### ISSUE THREAD

**Issue by Arunachala-Ramana** [pychess/pychess#1858]
The chessboard co-ordinates are very large (one can't distinguish between "E" file and "F" file clearly) and they seem to go out of the chessboard frame. Perhaps, it would be nice if the text size of the board coordinates are reduced. 
![puzzle](https://user-images.githubusercontent.com/68316708/87563706-29d33100-c6dd-11ea-9ff8-89880da472a9.png)

**Comment by hartwork** [comment]
It looks sane for me on Linux with Git `master`. I find no related settings. Was it fixed in the mean time?

**Comment by gbtami** [comment]
Yes, possible this was fixed before. If not, feel free to reopen.
